### PR TITLE
Docs: corrects outer join example

### PR DESCRIPTION
**Pull request by chri2547**
This PR corrects the outer join example. This issue was raised by a customer.

### CODE REVIEW

**Review by bohandley — CHANGES_REQUESTED** [review]
Just a little more detail since we're removing the query tables in the outer join description.

"While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field."

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 1]
```diff
@@ -359,32 +359,14 @@ The result after applying the inner join transformation looks like the following
 
 #### Outer join
 
-An outer join includes all data from an inner join and rows where values do not match in every input.
+An outer join includes all data from an inner join and rows where values do not match in every input. While the inner join joins on the time field, the outer join includes all rows that don’t match on the time field.
```
Comment (chri2547) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:362]:
```suggestion
An outer join includes all data from an inner join and rows where values do not match in every input. While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field.".
```

**Comment by chri2547** [comment]
> Just a little more detail since we're removing the query tables in the outer join description.
> 
> "While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field."

@bohandley , just made that change. Should I add the query tables back in? I thought that they were copy/paste error.

**Comment by bohandley** [comment]
> > Just a little more detail since we're removing the query tables in the outer join description.
> > "While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field."
> 
> @bohandley , just made that change. Should I add the query tables back in? I thought that they were copy/paste error.

I think it's wise to add them back in

> > Just a little more detail since we're removing the query tables in the outer join description.
> > "While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field."
> 
> @bohandley , just made that change. Should I add the query tables back in? I thought that they were copy/paste error.

Yep! I think keeping the tables in is a good idea.

```

#### Outer join

An outer join includes all data from an inner join and rows where values do not match in every input. While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field.".

In the following example, two queries return table data. It is visualized as two tables before applying the outer join transformation.

Query A:

| Time                | Job     | Uptime    |
| ------------------- | ------- | --------- |
| 2020-07-07 11:34:20 | node    | 25260122  |
| 2020-07-07 11:24:20 | postgre | 123001233 |
| 2020-07-07 11:14:20 | postgre | 345001233 |

Query B:

| Time                | Server   | Errors |
| ------------------- | -------- | ------ |
| 2020-07-07 11:34:20 | server 1 | 15     |
| 2020-07-07 11:24:20 | server 2 | 5      |
| 2020-07-07 11:04:20 | server 3 | 10     |

The result after applying the outer join transformation looks like the following:

| Time                | Job     | Uptime    | Server   | Errors |
| ------------------- | ------- | --------- | -------- | ------ |
| 2020-07-07 11:04:20 |         |           | server 3 | 10     |
| 2020-07-07 11:14:20 | postgre | 345001233 |          |        |
| 2020-07-07 11:34:20 | node    | 25260122  | server 1 | 15     |
| 2020-07-07 11:24:20 | postgre | 123001233 | server 2 | 5      |
```

**Comment by chri2547** [comment]
> > > Just a little more detail since we're removing the query tables in the outer join description.
> > > "While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field."
> > 
> > 
> > @bohandley , just made that change. Should I add the query tables back in? I thought that they were copy/paste error.
> 
> I think it's wise to add them back in
> 
> > > Just a little more detail since we're removing the query tables in the outer join description.
> > > "While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field."
> > 
> > 
> > @bohandley , just made that change. Should I add the query tables back in? I thought that they were copy/paste error.
> 
> Yep! I think keeping the tables in is a good idea.
> 
> ```
> 
> #### Outer join
> 
> An outer join includes all data from an inner join and rows where values do not match in every input. While the inner join joins Query A and Query B on the time field, the outer join includes all rows that don’t match on the time field.
> 
> In the following example, two queries return table data. It is visualized as two tables before applying the outer join transformation.
> 
> Query A:
> 
> | Time                | Job     | Uptime    |
> | ------------------- | ------- | --------- |
> | 2020-07-07 11:34:20 | node    | 25260122  |
> | 2020-07-07 11:24:20 | postgre | 123001233 |
> | 2020-07-07 11:14:20 | postgre | 345001233 |
> 
> Query B:
> 
> | Time                | Server   | Errors |
> | ------------------- | -------- | ------ |
> | 2020-07-07 11:34:20 | server 1 | 15     |
> | 2020-07-07 11:24:20 | server 2 | 5      |
> | 2020-07-07 11:04:20 | server 3 | 10     |
> 
> The result after applying the outer join transformation looks like the following:
> 
> | Time                | Job     | Uptime    | Server   | Errors |
> | ------------------- | ------- | --------- | -------- | ------ |
> | 2020-07-07 11:04:20 |         |           | server 3 | 10     |
> | 2020-07-07 11:14:20 | postgre | 345001233 |          |        |
> | 2020-07-07 11:34:20 | node    | 25260122  | server 1 | 15     |
> | 2020-07-07 11:24:20 | postgre | 123001233 | server 2 | 5      |
> ```

@bohandley , tables added back in. Thank you!

**Review by bohandley — APPROVED** [review]
LGTM!

**Pull request merged**